src/ClassApplication/Controllers/ClassController.cs

```diff
@@ -21,15 +21,23 @@ public ClassController(IHttpClientFactory factory, DatabaseService databaseServi
 
         /// <summary>
         /// 
-        /// Creates a classroom in the database from a given Classroom object
+        /// Creates a classroom in the database
         /// 
         /// </summary>
-        /// <param name="classroom">Has a ClassId, Name, and Description</param>
-        /// <returns>An array containing a value determined by the parameter</returns>
+        /// <param name="ownerId">ID of the user creating the class</param>
+        /// <param name="ownerName">Public name of the user creating the class</param>
         [HttpPost]
-        public async Task<ActionResult<string>> CreateClassroom(Classroom classroom)
+        public async Task<ActionResult<string>> CreateClassroom(string ownerId, string ownerName)

```

Comment:
Here's something to think about... do we want to give a person the option to add a description of their class right away or make them create the first? What do you think?